orange/src/main/java/com/hartwig/hmftools/orange/algo/purple/PurpleDataLoader.java

```diff
@@ -90,6 +94,8 @@ private static PurpleData load(@NotNull String tumorSample, @Nullable String ref
         List<StructuralVariant> allSomaticStructuralVariants =
                 StructuralVariantFileLoader.fromFile(somaticStructuralVariantVcf, new PassingVariantFilter());
 
+        List<StructuralVariant> allInferredSomaticStruturalVariants = loadInferredStructuralVariants(somaticStructuralVariantVcf);
```

Comment:
Feels a bit silly that we load an entire VCF twice. Can't we load once and then apply filtering?

Reply:
Yes this was bothering me also, but introducing inadvertent regressions while working on the basic implementation bothered me more 😄. I think worth improving, will have a look.